ynput/OpenPype
# Nuke Anamorphic slate - Read pixel aspect from input

## Changelog Description
When asset pixel aspect differs from rendered pixel aspect, Nuke slate pixel aspect is not longer taken from asset, but is readed via ffprobe.

## Testing notes:
1. Setup shot with 1828x1556, pixel aspect 2.0
2. Publish render from Nuke (with Nuke slate)
3. Change render output to be 1920x1080, pixel aspect 1.0
4. Publish again

With this PR, slate pixel aspect should not be squeezed in step 4

Might solve Issue #4614
